source/val/validate_image.cpp

```diff
@@ -455,13 +455,13 @@ spv_result_t ValidateImageOperands(ValidationState_t& _,
     }
 
     if (!_.options()->before_hlsl_legalization &&
-        spvIsVulkanEnv(_.context()->target_env)) {
+        spvIsVulkanEnv(_.context()->target_env) &&
+        !_.options()->allow_offset_texture_operand) {
       if (opcode != spv::Op::OpImageGather &&
           opcode != spv::Op::OpImageDrefGather &&
           opcode != spv::Op::OpImageSparseGather &&
           opcode != spv::Op::OpImageSparseDrefGather) {
         return _.diag(SPV_ERROR_INVALID_DATA, inst)
-               << _.VkErrorID(4663)
```

Comment:
Should this instead be [10213](https://registry.khronos.org/vulkan/specs/latest/html/vkspec.html#VUID-RuntimeSpirv-Offset-10213)?

Reply:
no, that is a "runtime" VU now, which means we would need to check for it at runtime if the extension is enabled or not

... honestly at this point, I feel this has just become inconsistent, but these VUID have always just been for tracking purpose, and I know things are all being covered here, so rather not spend too much time messing with VUID churn

Reply:
But in this case, the new option is effectively whether the feature is enabled no?

Reply:
ya, ok, you are right, let me just update this then, I guess felt strange adding a `runtime` vu to `spirv-val` list of VUs, but will just add a note its the exception, not the norm

Reply:
updated